extensions/notebook/src/jupyter/serverInstance.ts

```diff
@@ -143,11 +143,11 @@ export class PerFolderServerInstance implements IServerInstance {
 	}
 
 	private async configureJupyter(): Promise<void> {
-		await this.createInstanceFolders();
+		this.createInstanceFolders();
```

Comment:
If these methods are async, how do we know that the operations all complete before exiting the method, though?

Reply:
Ah.. my bad. I need to change the methods themselves to be synchronous.